test/CMakeLists.txt

```diff
@@ -43,3 +43,44 @@ compile_benchmark_test(cxx03_test)
 set_target_properties(cxx03_test
     PROPERTIES COMPILE_FLAGS "${CXX03_FLAGS}")
 add_test(cxx03 cxx03_test --benchmark_min_time=0.01)
+
+# Add the coverage command(s)
+if (${CMAKE_BUILD_TYPE} MATCHES "[Cc]overage")
```

Comment:
I would really like `COVERAGE` to work as well. Could you convert `${CMAKE_BUILD_TYPE}` to upper-case (or lower-case) before comparing?